### PR TITLE
PIM-4917: Change "float" type in price and metric mapping to "decimal"

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by jjanvier** [comment]
GTM

**Comment by nidup** [comment]
GTM once travis will be green :)

File: src/Pim/Bundle/CatalogBundle/Resources/config/comparators.yml [hunk 1]
```diff
@@ -55,6 +56,13 @@ services:
         tags:
             - { name: pim_catalog.attribute.comparator }
 
+    pim_catalog.comparator.attribute.number:
+        class: %pim_catalog.comparator.attribute.number.class%
+        arguments:
+            - ['pim_catalog_number']
+        tags:
+            - { name: pim_catalog.attribute.comparator, priority: -128 }
```
Comment (sumbobyboys) [inline src/Pim/Bundle/CatalogBundle/Resources/config/comparators.yml]: why this kind of priority ?

File: CHANGELOG-1.4.md [hunk 2]
```diff
@@ -30,6 +30,7 @@
 
 ## Bug fixes
 - PIM-4882: Cannot import products into a variant group if an axis is a reference data
+- PIM-4917: Add a number comparator
```
Comment (nidup) [inline CHANGELOG-1.4.md]: Maybe "Fix the issue with decimal storage for prices and number and add the related number comparator"

**Comment by nidup** [comment]
GTM

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**